src/components/style.css

```diff
@@ -90,6 +107,42 @@ body {
   font-size: 24px;
 }
 
+.linklist {
+  margin-left: 16px;
+  text-decoration: none;
+  padding: 6px;
+  color: #000;
+  font-size: 1.2rem;
+}
+
+.linklist:hover {
+  color: #5c3003;
+}
+
+.Home {
+  margin-top: 30px;
+  padding: 10px 20px;
+}
+
+.Home p {
+  margin-left: 100px;
+  margin-right: 100px;
+}
+
+.Home h2 {
+  margin-left: 60px;
+  font-size: 1.7em;
+  margin-bottom: 20px;
+}
+
+.Quote p {
+  margin-top: 100px;
+  margin-left: 100px;
+  margin-right: 100px;
+  padding: 20px 20px;
+  font-size: 1.7em;
+}
+
 .quote h1 {
   font-family: cursive;
   margin-left: 17rem;
```

Comment:
- [ ] Kindly make sure to center the Quotes page contents in order to match the wireframes.

  - ### YOUR DESIGN
  
   ![chrome-capture-2023-5-10](https://github.com/Enoisong/math-magicians/assets/87197412/6921471e-93cc-4585-a5cc-81bc6d09ed00)

  - ### WIREFRAME DESIGN
     ![image](https://github.com/Enoisong/math-magicians/assets/87197412/3d13ac8c-592d-4c46-a754-f12ad99c00d1)